lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -218,7 +219,7 @@ private void setTextFromAngle(int angleValue) {
     }
 
     private void initAttributes(TypedArray a) {
-        mColorWheelStrokeWidth = a.getInteger(
+        mColorWheelStrokeWidth = a.getDimension(
```

Comment:
Good catch but it should be `getDimensionPixesSize` which will make `mColorWheelStrokeWidth` an `int` rather than a `float`.